packages/sitecore-jss-nextjs/src/context/context.ts

```diff
@@ -1,3 +1,13 @@
+/**
+ * Page state enum
+ * Duplicates LayoutServicePageState to simplify future context migration into CloudSDK
+ */
+export enum PageStateType {
```

Comment:
I'm not sure that we need to introduce another _PageState_ type, we already have it under _@sitecore-jss/sitecore-jss/layout_: https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss/src/layout/models.ts#L23

Reply:
We do - but there's an assumption all of this code will be migrated into CloudSDK later on. Since we won't be able to reference sitecore-jss package from there, having this duplicate will simplify the migration.

Reply:
You are right, this code will be migrated, but we will also migrate our common types, sitecore-jss package will be splitted into _common_, _editing_ (may be?), etc.

Reply:
It's a bit early to think about migration, when we will migrate, we will update all the places accordingly